### PR TITLE
Install wizard

### PR DESCRIPTION
**Pull request by uldisrudzitis**
- PHP version check update
- Sync install wizard fields
- Sync berta install. Create and sync default `Home` section after install
- Remove outdated video tutorials

### CODE REVIEW

File: INSTALL/includes/first_visit_serverreqs.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 if(empty($CHECK_INCLUDED)) {
-	define('AUTH_AUTHREQUIRED', true);
+    define('AUTH_AUTHREQUIRED', true);
 	define('SETTINGS_INSTALLREQUIRED', false);
```
Comment (vteivans) [inline INSTALL/includes/first_visit_serverreqs.php]: why the difference in indent?

File: _api_app/app/Configuration/SiteSettingsConfigService.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 class SiteSettingsConfigService
 {
     private $settings;
+    private $defaultTemplate = 'messy-0.4.2';
```
Comment (vteivans) [inline _api_app/app/Configuration/SiteSettingsConfigService.php]: why not simply put this in the default value field in settings file? `engine/inc.settings.php:57`

Comment (uldisrudzitis) [reply _api_app/app/Configuration/SiteSettingsConfigService.php]: Already tried, old Berta doesn't generate `settings.xml` file in case it's defined.

Comment (vteivans) [reply _api_app/app/Configuration/SiteSettingsConfigService.php]: Then please add a comment why this is done here.

**Pull request merged**